selendroid-standalone/src/main/java/io/selendroid/standalone/SelendroidConfiguration.java

```diff
@@ -99,6 +99,10 @@
   @Parameter(names = "-maxInstances", description = "Maximum number of instances that a grid hub can use at a time.")
   private int maxInstances = 5;
 
+  @Parameter(names = "-registerCycle", description = "How often in ms the node will try to register itself again" +
```

Comment:
@asm89, thanks! Good point. 
Do you think that setting registerCycle to 0 would be sufficient to disable this re-register behavior? (just to not introduce any other flags)